src/license/licenseClient.ts

```diff
@@ -0,0 +1,13 @@
+import { AbstractClient, Parameters } from '../abstractClient';
+import { LicenseGetData } from './entities/getLicense/licenseGetResult';
+
+export class LicenseClient extends AbstractClient {
+  public getLicense(
```

Comment:
I think this method should be in `LicensesClient`
I think it would be more simple to have only 1 entry point to each API domain
`/catalog` => `CatalogClient`
`/licenses` => `LicensesClient`
`/customers` => `CustomersClient`

Reply:
Done